backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -199,7 +199,7 @@ const validateRecord = async (
   }
 }
 const blacklistIfNeeded = async (message: any): Promise<void> => {
-  const notificationType = message?.notificationType
+  const notificationType = message?.notificationType // should this be message?.notificationType || message?.eventType?
```

Comment:
`message?.eventType` is a subset of `message.notificationType` as we're using SNS. Could we remove this comment?